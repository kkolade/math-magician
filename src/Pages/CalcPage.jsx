import React from 'react';
import Calculator from '../components/Calculator';
import './calc-page.css';

function CalcPage() {
  return (
    <div className="page-content">
      <h2 className="headline">
        <span>Let</span>
        <span>&apos;</span>
        <span>s do some math!</span>
      </h2>
      <Calculator />
    </div>
  );
}

export default CalcPage;
