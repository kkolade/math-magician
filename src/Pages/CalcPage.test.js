import React from 'react';
import renderer from 'react-test-renderer';
import CalcPage from './CalcPage';

test('CalcPage component renders correctly', () => {
  const component = renderer.create(<CalcPage />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
