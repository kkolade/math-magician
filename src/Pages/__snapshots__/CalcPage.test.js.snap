// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalcPage component renders correctly 1`] = `
<div
  className="page-content"
>
  <h2
    className="headline"
  >
    <span>
      Let
    </span>
    <span>
      '
    </span>
    <span>
      s do some math!
    </span>
  </h2>
  <div
    className="calc-container"
  >
    <div
      className="display"
      data-testid="display"
    >
      0
    </div>
    <button
      className="calc-key"
      onClick={[Function]}
      type="submit"
    >
      AC
    </button>
    <button
      className="calc-key"
      onClick={[Function]}
      type="submit"
    >
      +/-
    </button>
    <button
      className="calc-key"
      onClick={[Function]}
      type="submit"
    >
      %
    </button>
    <button
      className="operator"
      onClick={[Function]}
      type="submit"
    >
      รท
    </button>
    <button
      className="calc-key"
      onClick={[Function]}
      type="submit"
    >
      7
    </button>
    <button
      className="calc-key"
      onClick={[Function]}
      type="submit"
    >
      8
    </button>
    <button
      className="calc-key"
      onClick={[Function]}
      type="submit"
    >
      9
    </button>
    <button
      className="operator"
      onClick={[Function]}
      type="submit"
    >
      x
    </button>
    <button
      className="calc-key"
      onClick={[Function]}
      type="submit"
    >
      4
    </button>
    <button
      className="calc-key"
      onClick={[Function]}
      type="submit"
    >
      5
    </button>
    <button
      className="calc-key"
      onClick={[Function]}
      type="submit"
    >
      6
    </button>
    <button
      className="operator"
      onClick={[Function]}
      type="submit"
    >
      -
    </button>
    <button
      className="calc-key"
      onClick={[Function]}
      type="submit"
    >
      1
    </button>
    <button
      className="calc-key"
      onClick={[Function]}
      type="submit"
    >
      2
    </button>
    <button
      className="calc-key"
      onClick={[Function]}
      type="submit"
    >
      3
    </button>
    <button
      className="operator"
      onClick={[Function]}
      type="submit"
    >
      +
    </button>
    <button
      className="zero"
      onClick={[Function]}
      type="submit"
    >
      0
    </button>
    <button
      className="calc-key"
      onClick={[Function]}
      type="submit"
    >
      .
    </button>
    <button
      className="operator"
      onClick={[Function]}
      type="submit"
    >
      =
    </button>
  </div>
</div>
`;
